crates/yttrium/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+uniffi::setup_scaffolding!();
```

Comment:
I think you need this?

```suggestion
#[cfg(feature = "uniffi")]
uniffi::setup_scaffolding!();
```